<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysChargeItem</class>
 <widget class="QWidget" name="SysChargeItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_insert">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>34</height>
          </size>
         </property>
         <property name="text">
          <string>新增</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_update">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>34</height>
          </size>
         </property>
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_delete">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>34</height>
          </size>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_refresh">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>34</height>
          </size>
         </property>
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>320</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>24</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>40</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
